src/data/finset/pairwise.lean

```diff
@@ -30,15 +30,24 @@ lemma pairwise_disjoint.elim_finset [decidable_eq α] {s : set ι} {f : ι → f
   i = j :=
 hs.elim hi hj (finset.not_disjoint_iff.2 ⟨a, hai, haj⟩)
 
-lemma pairwise_disjoint.image_finset_of_le [decidable_eq ι] [semilattice_inf α] [order_bot α]
-  {s : finset ι} {f : ι → α} (hs : (s : set ι).pairwise_disjoint f) {g : ι → ι}
+section semilattice_inf
+variables [semilattice_inf α] [order_bot α] {s : finset ι} {f : ι → α}
+
+lemma pairwise_disjoint.image_finset_of_le [decidable_eq ι] (hs : (s : set ι).pairwise_disjoint f)
```

Comment:
This argument order is a bit weird, normally the `decidable_eq` would appear before `s`